llama_hub/wordlift/requirements.txt

```diff
@@ -0,0 +1,4 @@
+langchain
```

Comment:
you don't need this, openai, or llama-index (all of these are installed as part of the core llama-index package which is already a dep for llama_hub)

Reply:
@jerryjliu Resolved